public/js/components/AtomEdit/CustomEditors/StoryQuestionFields/Question.js

```diff
@@ -27,21 +39,43 @@ export class StoryQuestionsQuestion extends React.Component {
     this.props.onUpdateField(questionWithId);
   }
 
-  render () {
-    const answersCount = (this.props.fieldValue && this.props.fieldValue.answers) ? this.props.fieldValue.answers.length : 0;
+  onSelect = (snippet) => {
```

Comment:
I can't understand how the answers in the props get updated if we're only maintaining `state.answers`

Reply:
I forgot to send the last two commits 😅

Reply:
oh, it looks like the code has updated in my browser since I wrote this comment. I can see how it works now!